### ISSUE TITLE
HTML tags are stripped from JSON when pushed to API 

### ISSUE THREAD

**Issue by nomad-mystic** [alleyinteractive/apple-news#352]
Hello Alley Interactive, 
I'm currently experiencing an issue with HTML tags being stripped from my JSON properties when the plugin sends them to the Apple News API. 

Attached is the JSON(txt) file from the download option in the plugin Articles page, the API response, and the setting page. (Version 1.2.4)

[article-1164071.txt](https://github.com/alleyinteractive/apple-news/files/920642/article-1164071.txt)
[api_call.txt](https://github.com/alleyinteractive/apple-news/files/920675/api_call.txt)
![apple_news_html_bug_screenshot](https://cloud.githubusercontent.com/assets/8921917/25021249/6675cf82-2046-11e7-849b-971497ed7b59.jpg)

Is there something I need to do to have my HTML tags make it to the Apple News API?

**Comment by bcampeau** [comment]
We're aware of this issue and we're looking into it. We're also going to fix an issue as part of this where empty paragraphs are being created when using HTML as described here: https://wordpress.org/support/topic/there-has-been-an-error-with-the-api-3/

**Comment by Samblis** [comment]
Thank you!

**Comment by nomad-mystic** [comment]
Thank you for taking on this issue in your next build. Keith

**Comment by superdav42** [comment]
We sure this is fixed in 1.2.7? I'm still getting:
```
NVALID_DOCUMENT - "text" was "" should be not (null or an empty string) (keyPath components->3->components->21->text)
INVALID_DOCUMENT - no components defined for role=body (keyPath components->3->components->21)
INVALID_DOCUMENT - "text" was "" should be not (null or an empty string) (keyPath components->3->components->23->text)
INVALID_DOCUMENT - no components defined for role=body (keyPath components->3->components->23)
```
After upgrading to 1.2.7
